src/services/ono/queries/projects/index.js

```diff
@@ -0,0 +1,88 @@
+
+export const GET_TOPLEVEL_PROJECTS_QUERY = `
+query {
+	repositories {
+    nodes {
+      id
+      name
+      owner {
+        fullName
+      }
+      description
+      updated
+    }
+  }
+  groups {
+    name
+    description
+  }
+}`
+
+export const GET_PROJECTS_QUERY = `
+query($name: String!) {
+	group(name: $name) {
+    repositories {
+      nodes {
+        name
+        id
+        owner {
+          fullName
+        }
+        description
+        updated 
+      }
+    }
+    groups {
+      name
+      description 
+    }
+  }
+}`
+
+
+export type ProjectType = {
+  name: string,
+  description?: string,
+  id: string,
+  owner: { fullName: string },
+  updatedTime: string,
+}
+
+export type GroupType = {
+  name: string,
+  description?: string,
+}
+
+export type ReturnType = {
+  projects: Array<ProjectType>,
+  groups: Array<GroupType>,
+}
+
+
+export function parseToplevelProjectsData(data: any): ReturnType {
+  return {
+    projects: data.data.repositories.nodes,
+    groups: data.data.groups,
+  }
+}
+
+function removeBaseName(item: Object, base: string) {
+  const next = item
+  next.name = item.name.replace(base, '')
+  return next
+}
+
+export function parseProjectsData(data: any, path: string): ReturnType {
```

Comment:
Is this logic needed for all projects fetched from the server? 

The `'Parent Group'` seems specific to a container/component. In which case it should be done in the container. If it isn't specific to a container/component, then it seems it should be done in the graphql endpoint.